something/BUILD.bazel

```diff
@@ -0,0 +1,3 @@
+exports_files([
```

Comment:
see previous comment for recommendation on credentials.json. Two main actions for you to do:
1) Please use the glob trick discussed. Having a `BUILD.bazel` file referencing a non-existent file is very likely to cause a build failure. We should have all build targets always succeed for ease of CI/CD pipeline.

2) Move the credentials directory somewhere else. Given that this is only for astore testing, I would put it somewhere under astore. Consider also that other parts of enkit, eg, the `proxy` in theory require credentials for testing, but different ones. So a top level credentials/ directory can end up being confusing, and require a global naming convention.

See other credentials directories scattered in the repo, also recommend adding a README, to indicate what isthe expected content of the directory.

Reply:
See above comment, it is a data dependency, but an absolute path not globbed.